code/modules/spells/aoe_turf/screech.dm

```diff
@@ -36,9 +36,14 @@
 			var/mob/living/carbon/human/H = C
 			if(H.earprot())
 				continue
-			if(!C.vampire_affected(user.mind))
-				continue
-			targets += C
+			var/success = C.vampire_affected(user.mind)
+			switch (success)
+				if (TRUE)
+					targets += C
+				if (FALSE)
+					continue
+				if (VAMP_FAILURE)
+					critfail(targets, user)
```

Comment:
Will screech the person with the rod too, add a return FALSE after that

Reply:
True